resources/views/headers/header.blade.php

```diff
@@ -49,13 +49,13 @@
 
 			</div>
 
-		@else
-
-			<div class="header__navigation">
-
-				<a href="{{ url('/login') }}">{{ trans('login.page_title')}}</a>
-
-			</div>
+//		@else
```

Comment:
@shenriod the location in the file is correct, but unfortunately the `//` is not a valid comment in the Blade files. The [Laravel Blade](https://laravel.com/docs/5.4/blade#comments) engine uses the fancier 
```
{{-- comment --}}
```

I would suggest here to delete the lines instead of commenting them